setup.cfg

```diff
@@ -24,7 +24,7 @@ classifiers =
 long_description_content_type = text/x-rst
 
 [options]
-python_requires = >=3.5.2
+python_requires = >=3.5.3
```

Comment:
why not bump to 3.5.4 ?
I see that this is the minimum version that we use for Azure and GHA testing.

MacOs and Linux are on 3.5.10 while Win is on 3.5.4.

If we decide to support 3.5.3 I think that we should at least update the GitHub Action workflow and in `actions/setup-python@v2` ask specifically for 3.5.4